viite-UI/src/view/navigationpanel/ProjectListModel.js

```diff
@@ -121,35 +122,52 @@
 
 
     var userFilterVisibility = function (showFilters) {
+      var searchBox = $('#userFilterSpan');
+      var textField = $('#userNameBox');
       if (showFilters) {
-        $('#userFilterSpan').show();
-        if ($('#userNameBox').val() === "") {
-          $('#userNameBox').val(applicationModel.getSessionUser());
+        searchBox.show();
+        if (textField.val() === "") {
+          textField.val(applicationModel.getSessionUser());
         }
-        filterByUser();
       } else {
-        $('#userFilterSpan').hide();
+        textField.val("");
+        searchBox.hide();
       }
+      filterByUser();
     };
 
     function bindEvents() {
 
       eventbus.once('roadAddressProjects:fetched', function(projects) {
-        projectArray = projects;
-        createProjectList(projects);
+        projectArray = _.filter(projects, function(proj) {
+          return proj.statusCode !== 7; //filter deleted projects out
```

Comment:
you can add the Project status code to the LinkValues.js in order to avoid magic numbers code ^^